pkgs/development/python-modules/ankipandas/default.nix

```
@@ -0,0 +1,48 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, numpy
+, pandas
+, colorlog
+, pytest
+, pytest-cov
+, randomfiletree
+}:
+
+buildPythonPackage rec {
+  pname = "ankipandas";
+  version = "0.3.15";
+  format = "setuptools";
+
+  src = fetchPypi {
+    inherit pname version;
+    hash = "sha256-31kkXytwUHeua6TBZMyfuvmQa9KNKcK/ielwDqNj37w=";
+  };
+
+  propagatedBuildInputs = [
```

consider using the new naming scheme for python deps.
```suggestion
  dependencies = [
```